docs/about-lisk/fees.md

```diff
@@ -108,8 +108,8 @@ weighted_gas_price = 16*base_fee_scalar*base_fee + blob_base_fee_scalar*blob_bas
 
 The current values for the scalars are:
 
-  - `base_fee_scalar` = 0.024740
-  - `blob_base_fee_scalar` = 0.801328
+  - `base_fee_scalar` = 37110
```

Comment:
I guess we should keep the 6 decimal precision format as in the previous version, meaning, 0.037110 and 1.201993